MAUI/Segmented-Control/display-mode.md

```diff
@@ -0,0 +1,215 @@
+---
+layout: post
```

Comment:
main title in list

Populating segment items

Reply:
changed title